egs/librimix/ConvTasNet/eval.py

```diff
@@ -87,6 +88,7 @@ def main(conf):
                 sf.write(local_save_dir + "s{}.wav".format(src_idx), src,
                          conf['sample_rate'])
             for src_idx, est_src in enumerate(est_sources_np):
+                est_src /= np.max(np.abs(est_src))  # to avoid clipping
```

Comment:
Then, you can multiply by the largest value in the mix. Doesn't make sense to have all the output have max of 1.